### PR TITLE
added edit form, refactored TemplateTask, fixed delete functionality

### PR DESCRIPTION
**Pull request by thamaranth**
Fixes # .

## Overview

(quick description of solution to original issue)

## Data Model / DB Schema Changes

(if there were data changes, explain them)
(also note whether it is necessary to run any database migrations)
(N/A if it doesnt apply)

## Environment / Configuration Changes

(any updated third-party libraries / packages?)
(any updates to the configuration files?)
(N/A if it doesnt apply)

## Notes

(heads up on anything not super intuitive or obvious)

### CODE REVIEW

File: src/routes/template_tasks.js [hunk 1]
```diff
@@ -26,6 +26,21 @@ router.post('/', function(req, res, next){
...
+  const { id } = req.params
+  const updateTemplateTask = {
+    title: req.body.title,
+    description: req.body.description,
+    days_to_complete: req.body.days_to_complete,
+    user_role: req.body.user_role
+  }
+  templateTask.update(id, updateTemplateTask)
+  .then(results => {
+    console.log(results)
```
Comment (punitrathore) [inline src/routes/template_tasks.js]: can you remove this please?

File: src/browser/components/atoms/List.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import React, { Component } from "react"
 import styles from './list.css'
+import ListItem from './ListItem/index'
+
```
Comment (punitrathore) [inline src/browser/components/atoms/List.js:4]: extra whitespace here

File: src/browser/components/atoms/listItem/index.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react"
+import TemplateTaskForm from '../../molecules/TemplateTaskForm'
+
+
```
Comment (punitrathore) [inline src/browser/components/atoms/listItem/index.js]: extra whitespace here

File: src/browser/components/atoms/listItem/index.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+               <button type="button" className="btn btn-default"
+                 aria-label="Left Align" onClick={this.toggleClickedEdit}>
+                 Edit
+               </button>
+             </td>
+           </tr>
+  }
+
+  render() {
+
```
Comment (punitrathore) [inline src/browser/components/atoms/listItem/index.js]: extra whitespace here

**Comment by thamaranth** [comment]
changes submitted.

**Pull request merged**